src/colorizer/CanvasOverlay.ts

```diff
@@ -63,18 +159,23 @@ export default class CanvasOverlay {
     }
   }
 
-  private renderScaleBar(): void {
-    if (!this.scaleBarOptions.unitsPerScreenPixel || !this.scaleBarOptions.visible) {
-      return;
-    }
+  // NOTE: Rendering of the scale bar and timestamp is deferred here, since we need to draw a background behind them
+  // with an appropriate size. These methods return a callback that renders the scale bar or timestamp to the canvas,
```

Comment:
what methods return a callback?

Reply:
not sure if this comment is adding value... since RenderInfo is pretty clear as to what it does.  Things seem to be named in an understandable way.